### ISSUE TITLE
Defaulting to Cha for HP/Fort saves, like undead

### ISSUE THREAD

**Issue by AlphaQup2nite** [Lavaeolous/PF1-StatBlock-Converter-Module#363]
Currently the SBC is treating everything like it is undead, defaulting to using Cha as the mod for HP and fort saves for everything.

**Comment by AlphaQup2nite** [comment]
this does not seem to be happening anymore.

**Comment by AlphaQup2nite** [comment]
ok, it seems to occur after creating Undead using the SBC.  It will get stuck in CHA default for everything coming afterwards.

**Comment by Lavaeolous** [comment]
Yeah, found the error. SBC checks for specific conditions while parsing the statblock and saves these locally.

```JS
"flags": {
        "noStr": false,
        "noDex": false,
        "noCon": false,
        "noInt": false,
        "noWis": false,
        "noCha": false,
        "isUndead": false   
    }
```

I'll have to include a reset for this flags.

**Comment by Lavaeolous** [comment]
Fixed in 3.0.1
